Gemfile.lock

```diff
@@ -405,69 +408,70 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.12.0)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.2)
     marco-polo (2.0.1)
       railties (>= 6)
-    maxitest (3.3.0)
-      minitest (>= 5.0.0, < 5.12.0)
-    metaclass (0.0.4)
-    method_source (0.8.2)
-    mime-types (3.4.1)
+    maxitest (3.7.0)
+      minitest (>= 5.0.0, < 5.15.0)
+    method_source (0.9.2)
+    mime-types (3.5.2)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2023.0218.1)
+    mime-types-data (3.2024.0206)
     mini_mime (1.1.5)
     mini_portile2 (2.8.5)
     minitest (5.11.3)
-    minitest-rails (6.1.0)
+    minitest-rails (6.1.1)
       minitest (~> 5.10)
       railties (~> 6.1.0)
     mixlib-shellout (3.2.7)
       chef-utils
-    mocha (1.2.1)
-      metaclass (~> 0.0.1)
+    mocha (1.16.1)
     momentjs-rails (2.29.4.1)
       railties (>= 3.1)
-    msgpack (1.4.2)
+    msgpack (1.7.2)
     multi_json (1.15.0)
     multi_xml (0.6.0)
     multipart-post (2.3.0)
-    mysql2 (0.5.3)
+    mysql2 (0.5.5)
     net-http-persistent (4.0.2)
       connection_pool (~> 2.2)
-    net-ldap (0.18.0)
+    net-ldap (0.19.0)
```

Comment:
### [net-ldap](http://github.com/ruby-ldap/ruby-net-ldap)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.18.0 → 0.19.0

[_~~change-log~~, [source-code](https://github.com/ruby-ldap/ruby-net-ldap), [gem-diff](https://my.diffend.io/gems/net-ldap/0.18.0/0.19.0)_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/ruby-ldap/ruby-net-ldap/compare/v0.18.0...v0.19.0).

These are the first 10 commits:
- (9c8525c) [Add in ability for users to specify LDAP controls when condu…](https://github.com/ruby-ldap/ruby-net-ldap/commit/9c8525c1b852ffc0bd62295195bb711022527d50)
- (e896715) [Fix using blank? since that might not exist, and also allow …](https://github.com/ruby-ldap/ruby-net-ldap/commit/e896715eee5e2f85be6e7211e6813044ba457d9d)
- (0055734) [Retain spaces in RDN values in DNs](https://github.com/ruby-ldap/ruby-net-ldap/commit/005573458081886edb8384ad43a2ae23715a1b42)
- (6b18901) [Merge branch 'master' into add-in-ldap-control-flexibility](https://github.com/ruby-ldap/ruby-net-ldap/commit/6b18901ecc5e154cbb42ec62020af3e4b096bd5e)
- (b53edf3) [Merge branch 'master' into retain_spaces_in_RDN_values](https://github.com/ruby-ldap/ruby-net-ldap/commit/b53edf36b3d66d346fde603ab30786e1a8da0007)
- (3d03e60) [Merge pull request #412 from TomSellers/retain_spaces_in_RDN…](https://github.com/ruby-ldap/ruby-net-ldap/commit/3d03e60407bc537f004bad671b06c3d4550afc88)
- (e7896d8) [Document `connect_timeout` in Constructor Details](https://github.com/ruby-ldap/ruby-net-ldap/commit/e7896d830f01a1984f0b4b21fea20a012092d52d)
- (d2d500b) [Add in tests](https://github.com/ruby-ldap/ruby-net-ldap/commit/d2d500b12b25b9bf8714c683b253fc57bbfaddd8)
- (03ab921) [Merge pull request #411 from gwillcox-r7/add-in-ldap-control…](https://github.com/ruby-ldap/ruby-net-ldap/commit/03ab921a054e80fa9e0a2ab10bbc59f5da5cdd29)
- (06acd16) [Update rubocop todo](https://github.com/ruby-ldap/ruby-net-ldap/commit/06acd16a09d5edbdfe8876de1e12503c571a4381)


</details>